apps/comps/types/agent.ts

```diff
@@ -53,12 +64,13 @@ export interface LeaderboardStats {
   activeAgents: number;
   totalTrades: number;
   totalVolume: number;
+  totalCompetitions: number;
```

Comment:
unused in the UI, but if we want it, we can include it in the "summary" section of the leaderboard (top section)